### ISSUE TITLE
Flash.Patcher.Exe detected as Virus by Microsoft's Windows Security

### ISSUE THREAD

**Issue by 00DogeCubeGamer00** [KuromeSan/FlashPatcher#8]
Everytime I try to download the file, Google and other websites automatically block it as a virus with it claiming to be this that's severe and the file is automatically deleted as unsafe.

![Microsoft Antivirus Issue](https://user-images.githubusercontent.com/79983357/109812497-c1947980-7bf1-11eb-951e-676bc4147385.png)

Can you please look into this and have Microsoft not flag this or fix somehow because I only use microsoft's windows security and the problem with websites blocking the download period and unable to install because of this supposing virus that exists.

**Comment by KuromeSan** [comment]
i just resolved this like 3 minutes ago in v1.6, so try that one ..

**Comment by 00DogeCubeGamer00** [comment]
I am trying v.1.6 by downloading it and it's still flagging it somehow with "Ulthar.A!ml severe trojan" and blocking it with websites so I am not sure what this problem is, tried Google Chrome and Microsoft Edge.

I have everything updated as well from Windows 10, so I don't know what the problem again is. 

So far, not downloadable, I have no clue what else to say or state or know what's going on.

**Comment by KuromeSan** [comment]
huh? i get nothing when i try download it ...

**Comment by Unlimited2018** [comment]
On my PC, FlashPatcher version 1.6 is no longer detected by Malwarebytes and 360 Total Security, indeed Edge blocks the download. I think I'm reporting it for SAFE to Microsoft, It's weird on Total Virus, Malwarebytes finds it as malware but the one on my PC doesn't say the same thing. What SecureAge APEX, Sangfor Engine Zero, Ikarus say doesn't really matter. Now the impression is blocked by Microsoft and it detects Malwarebytes that we can't use Flash Player. FlashPatcher is a good project and only heat using Flash Player. I tried other alternatives to Flash Player but a lot of software, games do not run. Thanks KuromeSan!

**Comment by 00DogeCubeGamer00** [comment]
It appears that they've changed the trojan name after an update or whatever.

It's now listed as this and still not fixed and unblocked from browsers:
![Another Microsoft Antivirus Issue](https://user-images.githubusercontent.com/79983357/109845632-f9f87f80-7c12-11eb-886e-9236baa0da7b.png)

- DCG.

**Comment by Unlimited2018** [comment]
Now Microsoft (Trojan:Win32/Wacatac.B!ml) and AVG (Avast), Cynet,  are detecting on Virustotal, I don't understand anything anymore.

**Comment by KuromeSan** [comment]
MS may not want to mark a program that edits windows system32 files as 'safe' tbh, but meh could  just mark it as "HackTool.FlashPatcher" so everyone knows to ignore it instead of some random trojan that changes w every av u use lol


mm that said- ive acturally had programs i made come up as Wacatac.B!ml many times before, 
submitting them to MS has worked everi time..

**Comment by Unlimited2018** [comment]
You are right even Blender has been detected as Trojan: Win32 / Wacatac.B! Ml   https://github.com/DotBow/Blender-Launcher/issues/43  That also surprised me why I don't tell the truth "HackTool.FlashPatcher"   or "Not a virus, HackTool"

**Comment by Unlimited2018** [comment]
Now at Microsoft  is no more  Trojan: Win32 / Wacatac.B! Ml but is Program: Win32 / Wacapew.C! Ml  :)

**Comment by 00DogeCubeGamer00** [comment]
What's next, wacauthashoot.D!ml? What's next, wacatacapewathaula.E!ml, I can't wait to see the D and E letters in the alphabet, thanks Microsoft, you're the best.

**Comment by crackerjack121** [comment]
it's not a virus even mcaffe said it was disable live scan run  defuse then reenable  all you do i use the opera gx browser and it worked flawless picture as proof
![Capture](https://user-images.githubusercontent.com/80162681/110210086-b0e43d80-7e55-11eb-9d42-6d9782cf68e7.PNG)

**Comment by KuromeSan** [comment]
i gave the file to microsoft, lets see if they do anything - https://www.microsoft.com/en-us/wdsi/submission/3f371e7c-0966-4641-8a0b-d726d52ff823

**Comment by LiEnby** [comment]
it is now no longer Win32/Trojan!Wactac.B!ml and is now "Hacktool/Patcher" which is *not* a virus detection-

**Comment by LiEnby** [comment]
![image](https://user-images.githubusercontent.com/39113159/110430017-9c7f9980-8110-11eb-91e7-e5dd8a9d68ed.png)
thats the best MS could give me i guess- haha

**Comment by LiEnby** [comment]
https://forums.malwarebytes.com/topic/271519-flashpatcher-detected-as-genericmalwaresuspicious/ malwarebytes are going to go the same way it seems-

**Comment by aspdotnut** [comment]
It is now being detected as Trojan:Win32/AgentTesla!ml.
![image](https://user-images.githubusercontent.com/55002387/110926572-daeda200-8324-11eb-87be-f7e70a8defe2.png)

**Comment by LiEnby** [comment]
update your virus definitions ..
